ui/src/plugins/dev.perfetto.WidgetsPage/demos/nodegraph_demo.ts

```diff
@@ -1235,6 +1269,60 @@ export function NodeGraphDemo(): m.Component<NodeGraphDemoAttrs> {
             }
           });
         },
+        labels: store.labels,
+        selectedLabelIds: selectedLabelIds,
+        onLabelMove: (labelId: string, x: number, y: number) => {
```

Comment:
Looks like onLabelMove and onLabelResize are being called on every mouse move event - this will cause issues with undo/redo and is generally undesirable. Maybe just call these callbacks on mouseup